lib/ansible/modules/cloud/azure/azure_rm_acr.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Yawei Wang, <yaweiw@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_acr
+version_added: "2.5"
+short_description: Manage an Azure Container Registry (ACR).
+description:
+    - Create, update and delete an Azure Container Registry.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Container Registry exists or will be created.
+        required: true
+    name:
+        description:
+            - Name of the Container Registry.
+        default: null
+        required: true
+    state:
+        description:
+            - Assert the state of the ACR. Use 'present' to create or update an ACR and 'absent' to delete it.
+        default: present
+        choices:
+            - absent
+            - present
+        required: false
+    location:
+        description:
+            - Valid azure location. Defaults to location of the resource group.
+        default: resource_group location
+        required: false
+    admin_user_enabled:
+        description:
+            - If enabled, you can use the registry name as username and admin user access key as password to docker login to your container registry.
+        default: False
+        required: false
+    sku:
+        description:
+            - Specifies the SKU to use. Currently can be either Basic, Standard or Premium.
+        default: Standard
+        choices:
+            - Basic
+            - Standard
+            - Premium
+        required: false
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Yawei Wang (@yaweiw)"
+
+'''
+
+EXAMPLES = '''
+    - name: Create an azure container registry
+      azure_rm_acr:
+        name: testacr1
+        location: eastus
+        resource_group: testrg
+        state: present
+        admin_user_enabled: True
```

Comment:
same here

Reply:
tested with True or true. all worked. will go with lower case for consistence.